index.html

```diff
@@ -1,12 +1,28 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" ng-app='comicsContactsApp'>
 <head>
   <meta charset="UTF-8">
   <title>Superhero Phone Book</title>
   <link rel="stylesheet" href="style.css">
 </head>
-<body>
 
+<body ng-controller="comicsContactsController as vm">
+  <h2>Search Contacts</h2>
+  <input type="text" ng-model="searchSuperHeroes.names">
+  <hr/>
+
+  <div ng-repeat="superHero in vm.superHeroes | filter: searchSuperHeroes.names" ng-class="{ 'imageColor' : superHero.affiliation === 'Image Comics', 'marvelColor' : superHero.affiliation === 'Marvel Comics' }">
+    <img ng-src='{{superHero.image}}' ng-show="viewImage"/>
+
```

Comment:
better form is to have viewImage as a property of `vm` so that you aren't binding to a primitive (boolean) `vm.viewImage`